clang/test/3C/b30_structprotocastexplicitunsafeuseunsafe.c

```diff
@@ -62,16 +63,16 @@ struct np *bar() {
   x->next = &y;
   y->next = &x;
   struct np *z = (struct np *) sus(x, y);
-	//CHECK: struct np *z = (struct np *) sus(x, y);
+	//CHECK: struct np *z = (struct np *) sus(_Assume_bounds_cast<_Ptr<struct r>>(x), _Assume_bounds_cast<_Ptr<struct r>>(y));
   z += 2;
   return z;
 }
```

Comment:
This is one of those places where I'm unsure. It looks complicated so it must be

.....right? (Edit: If this test seems weird, all the b30* tests are like this and include diffs like this, so maybe take a look at the if those help)